src/refract/RenderJSONVisitor.cc

```diff
@@ -106,6 +107,16 @@ namespace refract
         return TypeQueryVisitor::as<T>(*(a->value.begin()));
     }
 
+    template<typename T>
+    const T* getNullable(const T& e) {
+        IElement::MemberElementCollection::const_iterator ta = e.attributes.find("typeAttributes");
+        
+        // from within ta we have to look for the 'nullable' attribute
+        
+        // this should return the new value if nullable
+        return NULL;
```

Comment:
This might be tricky since the other functions return null or the value, however for us the value is null. Perhaps I will return true and then in getValue return null when this returns true?